queue_fetcher/utils/sqs.py

```diff
@@ -116,19 +116,19 @@ def send_message(queue, message, raise_exception=True):
     except AttributeError:
         raise ImproperlyConfigured(SQS_NOT_SETUP)
 
+    is_text = (isinstance(message, six.string_types)
+               or isinstance(message, six.binary_type))
```

Comment:
`six.string_types` is the same as `(six.text_type, six.binary_type)`

Reply:
```
>>> isinstance(b'hello', six.string_types)
False
```